packages/ui/booker/DatePicker.tsx

```diff
@@ -87,50 +88,50 @@ const Days = ({
 
   return (
     <>
-      {days.map((day, idx) => (
-        <div key={day === null ? `e-${idx}` : `day-${day}`} className="relative w-full pt-[100%]">
-          {day === null ? (
-            <div key={`e-${idx}`} />
-          ) : (
-            <DayComponent
-              date={day}
-              onClick={() => props.onChange(day)}
-              disabled={
-                (includedDates && !includedDates.includes(yyyymmdd(day))) ||
-                excludedDates.includes(yyyymmdd(day)) ||
-                day.valueOf() < minDateValueOf
-              }
-              active={selected ? yyyymmdd(selected) === yyyymmdd(day) : false}
-            />
-          )}
-        </div>
-      ))}
+      {days.map((day, idx) =>
+        day === null ? (
+          <div key={`e-${idx}`} />
+        ) : (
+          <div key={day === null ? `e-${idx}` : `day-${day}`} className="relative w-full pt-[100%]">
+            {day === null ? (
+              <div key={`e-${idx}`} />
+            ) : props.isLoading ? (
+              <SkeletonContainer>
+                <button
+                  disabled
+                  className=" absolute top-0 left-0 right-0 bottom-0 mx-auto w-full rounded-sm border-transparent bg-gray-200 text-white"
```

Comment:
have you tested this on dark mode?

Reply:
😜

https://user-images.githubusercontent.com/8019099/175287120-c97e8842-c7fa-440f-9c84-5e6c4b528cfc.mov